### ISSUE TITLE
Training the model produces noise (for ZH)

### ISSUE THREAD

**Issue by RaduBolbo** [SWivid/F5-TTS#504]
### Checks

- [X] This template is only for usage issues encountered.
- [X] I have thoroughly reviewed the project documentation but couldn't find information to solve my problem.
- [X] I have searched for existing issues, including closed ones, and couldn't find a solution.
- [X] I confirm that I am using English to submit this report in order to facilitate communication.

### Environment Details

Ubuntu LTS 24, python 3.11

### Steps to Reproduce

0. In the _trainer.py_ script add code to save more audio wav files instead of just one wav file. For example, i save 10 reference and 10 generated files: like I do in this code with _save_current_model_sample() method [trainer.txt](https://github.com/user-attachments/files/17845513/trainer.txt)
1. run the train.py script on a dataset with both En and Zh data and the hyperparameters: batch_size_type = "frame"; batch_size_per_gpu = 384 ; num_warmup_updates = 300; learning_rate = 7.5e-5; epochs=1 (The rest of parameters take the default values) => from here **model1** is the resulted model
2. Run the train.py script again, but with learning_rate = 1e-5; epochs=5 (rest of hyperparameters set as in 1.) => from here **model1** is the resulted model
3. Listen to the English samples: for **model1** the English audio should begin to sound worse and gibberishworse () as the model is trained, but **model2** has a performance that remains roughly constant. I guess this has to do with the learning rate. I mention that the training starts from the pretrained **model_1200000**
4. Listen to the Chinese samples: both models have issues with Zh, as the generated audios either start with noise or are completely noise. This is not only true with the model after some training, but also with the very first iteration when the models hasn't actually been updated: so the pretraiend **model_1200000** is used (make sure that audio samples saving enables you to see that)
5. Now use run the api.py by using eventually the same audio samples and the **model_1200000** , **model2**  and **model2** . The results of the 3 models are similar: the issues with English disappears for **model1** and the issue with noise in Chinese disappears for **model_1200000** (I have changed the path in "ckpt_file = str(cached_path("hf://SWivid/F5-TTS/F5TTS_Base/model_1200000.safetensors"))" in the api.py script to modify the model).

### ✔️ Expected Behavior

1. The audio files saved during training for **model1** and **model2** should be intelligible and natural fro both En and Zh like the pretrained model is and the performance shouldn't drop by a lot as the model begin to be optimized.
2. The Zh results for the saved audio samples at the very beginning of the training should be intelligible, because the **model_1200000** has not begin to be optimized yet
3. If the results for 1. and 2. are poor and exhibit the described behavior, than the same behavior should also be present during inference with the api.py script when the same models are used.

### ❌ Actual Behavior

1. for **model1** the English audio should begin to sound worse and worse as the model is trained like gibberish, but **model2** has a performance that remains roughly constant.
2. both models have issues with Zh, as some audios either start with noise or are completely noise even at the very beginning of the training
3. The bad behavior of the models disappears if it is loaded in the api.py script: both Zh and En texts are uttered clearly and naturally for all the models. I believed it bight be because of the EMA use during inference, but it persists even without using EMA
4. Why are **api.py inference** and **the inference for obtaining the sample audios during training** so different?
5. Do you think that 7.5e-5 is too high for training further on the pretrained **model_1200000**?

**Comment by SWivid** [comment]
not quite get your main points

so how is the output, any examples to analyze? and what is the ref_audio _text and gen_text used

**Comment by SWivid** [comment]
will close this issue, feel free to open if further questions
